src/I18n/FrozenTime.php

```diff
@@ -98,6 +98,13 @@ class FrozenTime extends Chronos implements JsonSerializable
     public static $wordEnd = '+1 month';
 
     /**
+     * serialise the value as a Unix Timestamp
+     *
+     * @var string
```

Comment:
We shouldnt add @ var type for constants. This makes no sense because the type by definition is clear from the value. IDEs also completely ignore it due to that fact.